components/ContactUs/index.vue

```diff
@@ -25,7 +25,11 @@
 
 <script>
 export default {
-
+  methods: {
+    onClickCTA () {
+      window.open('https://api.whatsapp.com/send?phone=6281312848205', '_blank')
```

Comment:
**discussion**:
which one is better? write the phone number in code or move it to variable?
fyi it can be simplified to `https://wa.me/6281312848205` (optional)

Reply:
nice information thankyou 😄